java/ql/lib/ext/java.util.zip.model.yml

```diff
@@ -4,4 +4,11 @@ extensions:
       extensible: summaryModel
     data:
       - ["java.util.zip", "GZIPInputStream", False, "GZIPInputStream", "", "", "Argument[0]", "Argument[this]", "taint", "manual"]
+      - ["java.util.zip", "ZipEntry", True, "ZipEntry", "(String)", "", "Argument[0]", "ReturnValue", "taint", "ai-manual"]
       - ["java.util.zip", "ZipInputStream", False, "ZipInputStream", "", "", "Argument[0]", "Argument[this]", "taint", "manual"]
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["java.util.zip", "ZipFile", True, "getEntry", "(String)", "", "Argument[0]", "path-injection", "ai-manual"]
```

Comment:
`ZipFile.getEntry` seems to be producing some FPs. At the end of the day, any path traversal one could smuggle in here would be restricted to the ZIP root anyway, which seems safe enough. Exploiting access to arbitrary contents of a ZIP file is probably too context-dependent to be generally considered a sink. I'd recommend removing this row:

```suggestion
```